tests/pytorch_pfn_extras_tests/runtime_tests/test_jit_runtime.py

```diff
@@ -0,0 +1,97 @@
+import types
+import warnings
+
+import torch
+import torch.nn.functional as F
+
+import pytorch_pfn_extras as ppe
+from pytorch_pfn_extras.onnx._as_output import trace
+
+
+class JITRuntime(ppe.runtime.PyTorchRuntime):
+
+    def move_module(self, module):
+
+        def new_forward(self, *args):
+            if hasattr(self, '_traced_mod'):
+                out = self._traced_mod(*args)
+                inter_size = len(self._names)
+                if inter_size == 0:
+                    return out
+                if not isinstance(out, (tuple, list)):
+                    out = [out]
+                return dict(
+                    **{str(i): x for i, x in enumerate(out[:-inter_size])},
+                    **{name: x for name, x in zip(self._names, out[-inter_size:])},
+                )
+
+            new_forward = self.forward
+            self.forward = self._orig_forward
+
+            with trace(self) as (new_module, outputs):
+                with warnings.catch_warnings():
+                    warnings.simplefilter("ignore")
+                    self._traced_mod = torch.jit.trace_module(
```

Comment:
For record:
* Due to PyTorch bug, this raises `SystemError` when `filterwarnings = error` is set in `pytest.ini`.
* Tried to add `ignore::torch.jit.TracerWarning:torch.(?!jit)` to the filter to emulate https://github.com/pytorch/pytorch/blob/v1.10.0/torch/jit/_trace.py#L531-L540, however this broke EnsureShape JIT test for some reason. (#430)
* `catch_warnings` is thread-unsafe but we accept it for now as we don't have a plan to use pytest-xdist.